mon-pix/app/components/tutorial-item.js

```diff
@@ -42,12 +43,16 @@ export default class TutorialItemComponent extends Component {
     return this.evaluationStatus === buttonStatusTypes.recorded;
   }
 
-  get buttonText() {
-    return this.savingStatus === buttonStatusTypes.recorded ? 'Retirer' : 'Enregistrer';
+  get buttonContent() {
+    return this.savingStatus === buttonStatusTypes.recorded ?
+      this.intl.t('pages.user-tutorials.tutorial.actions.remove.title') :
+      this.intl.t('pages.user-tutorials.tutorial.actions.save.title');
   }
```

Comment:
Partie de l'autre commentaire : 
```suggestion
  get buttonContent() {
    return this.savingStatus === buttonStatusTypes.recorded ?
      this.intl.t('pages.user-tutorials.tutorial.actions.remove.content') :
      this.intl.t('pages.user-tutorials.tutorial.actions.save.content');
 }
```